sdk/resourcemanager/Azure.ResourceManager/api/Azure.ResourceManager.netstandard2.0.cs

```diff
@@ -255,7 +255,7 @@ public partial class ManagementGroupNameAvailabilityOptions
     {
         public ManagementGroupNameAvailabilityOptions() { }
         public string Name { get { throw null; } set { } }
-        public string ResourceType { get { throw null; } set { } }
```

Comment:
The only supported type is [Microsoft.Management/managementGroups](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/managementgroups/resource-manager/Microsoft.Management/stable/2021-04-01/management.json#L1939) , so it's also `ResourceType`.